README.md

```diff
@@ -179,6 +179,10 @@ To build an exchange container with code that is targeted for a git branch:
     - If maxAgreements>1, for CS, in search don't return node to agbot if agbot from same org already has agreement for same service.
     - Consider changing all creates to POST, and update (via put/patch) return codes to 200
 
+## Changes in 1.108.0
```

Comment:
Should we add something here about the translation files being added?

Reply:
let's do that in a separate PR